audio/audio.py

```diff
@@ -76,36 +64,12 @@ def apply_tfms(self, tfms):
     @property
     def shape(self): return self.data.shape
 
-    def _reload_signal(self):
```

Comment:
All of this code had a purpose, although there may be a better way to do it that I'm not aware of.

`_reload_signal` is here so that when the user calls `.show_batch()` or views `AudioItem`s in some way that shows the signal and spectrogram, the signal will be reloaded. Otherwise we have to load it every time we grab an item for training even though the audio isn't being used.

The internal variables `_sig` and `_sr` were being used to assist with that lazy loading.

Reply:
Ok, that definitely makes sense now that I've tried reimplementing lazy loading.  I had almost an identical solution so I tried changing the names back and only using `_sig` and `_sr` to get `sig` and `sr` populated